weather_mv/loader_pipeline/sinks_test.py

```diff
@@ -38,6 +43,32 @@ def decorated(*args, **kwargs):
     return decorated
 
 
+def generate_dataset() -> str:
```

Comment:
Would you mind generating the data with Xarray? It should have the ability built-in to write netCDF files.